### ISSUE TITLE
Opacity slider doesn't work for custom biomes

### ISSUE THREAD

**Issue by Fornad95** [Captain-Chaos/WorldPainter#359]
As per title - the slider works for custom annotations but not for custom biomes

**Comment by Captain-Chaos** [comment]
Yes, that's a feature, not a bug. Custom biomes were already transparent and are rendered completely differently from other layers, so its non trivial to extend the configurable opacity support to it.

I'll change this issue to a feature request. How important is it? Why do you want to change the opacity of custom biomes? Usually people disable showing the biomes altogether when not working on them since it clutters up the editor view a lot.

**Comment by Fornad95** [comment]
> I'll change this issue to a feature request. How important is it? Why do you want to change the opacity of custom biomes? Usually people disable showing the biomes altogether when not working on them since it clutters up the editor view a lot.

I also usually disable the biomes when working. However, for my Middle-earth map, I am trying to create an accurate representation of grass/vegetation colours from a northerly latitude (approx. Scandinavia) down to a southerly latitude (approx. Tunisia) at a certain time of year. This is something that I can't really determine in-game because these changes occur over thousands of blocks and therefore I need the satellite view that WorldPainter provides.

At the moment, the transparent nature of custom biomes means I can't get an accurate colour representation of the grass, and therefore I will really struggle to do this. I looked at using custom annotations instead (which do have an opacity slider) but the problem with those is that they overlap (unlike normal annotations). Of course you can only have one biome on one block, so custom annotations won't work either.

**Comment by Fornad95** [comment]
I have found a workaround for this and therefore this issue can be closed if nobody else is interested in it.
